transformer_engine/pytorch/attention.py

```diff
@@ -1031,23 +1040,58 @@ def forward(self, max_seq_len: int, offset: int = 0):
 
 
 class FusedRoPEFunc(torch.autograd.Function):
+    """
+    Function for FusedRoPE
+
+    This implementation assumes the input tensor to be in `sbhd`, `bshd` or `thd` format and
+    the RoPE tensor to be of shape (s, 1, 1, d). It accepts arbitrary memory layouts to avoid
+    the expensive `.contiguous()` calls, thus it may not achieve the best memory access pattern.
+    """
+
     @staticmethod
     def forward(
-        ctx, t: torch.Tensor, cos_: torch.Tensor, sin_: torch.Tensor
+        ctx,
+        t: torch.Tensor,
+        freqs: torch.Tensor,
+        transpose_output_memory: bool = False,
+        qkv_format: str = "sbhd",
```

Comment:
`tensor_format` is more descriptive than `qkv_format` and is consistent with `apply_rotary_pos_emb`:
```suggestion
        tensor_format: str = "sbhd",
```

Reply:
Changed accordingly.